developing/cli/install.md

```diff
@@ -2,13 +2,56 @@
 
 ## Minimum Requirements
 
-The minimum recommended specs for running osmosisd is as follows:
+The minimum recommended specs for running a full osmosisd node is as follows:
 - 8-core (4 physical core), x86_64 architecture processor
-- 32 GB RAM (or equivalent swap file set up)
+- 32 GB RAM (or equivalent swap file set up); 16 GB also works
 - 1 TB of storage space
 
+You can check if you have enough memory to run osmosisd here: https://quicksync.io/networks/osmosis.html. 
 
-## Update System
+
+For most devices, not having the above specs is perfectly fine. Only around 65GB is required to pruned-sync a fullnode and testing has been done succesfully with a 16GB RAM Macbook Pro. 
```

Comment:
I would remove this since the storage requirement is fluid (when state synced it could be 8GB and then if not state synced it could increase >100GB).

Reply:
All comments are great - accepted & implementing